<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="chienpt-w7" name="TestTeamCalendar" tests="11" failures="1" timestamp="17 May 2016 09:22:55 GMT" time="124.768" errors="0">
  <testcase name="TC01" time="3.414" classname="com.vsii.cstar.testcase.TeamCalendar"/>
  <testcase name="TC02" time="34.034" classname="com.vsii.cstar.testcase.TeamCalendar"/>
  <testcase name="TC03" time="2.3" classname="com.vsii.cstar.testcase.TeamCalendar">
    <failure type="java.lang.AssertionError" message="expected [4198534946] but found [4198534946x:1]">
      <![CDATA[java.lang.AssertionError: expected [4198534946] but found [4198534946x:1]
	at com.vsii.cstar.testcase.TeamCalendar.TC03(TeamCalendar.java:459)
	at com.vsii.cstar.app.Main.main(Main.java:32)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- TC03 -->
  <testcase name="TC04" time="34.809" classname="com.vsii.cstar.testcase.TeamCalendar"/>
  <testcase name="TC05" time="2.841" classname="com.vsii.cstar.testcase.TeamCalendar"/>
  <testcase name="TC06" time="15.802" classname="com.vsii.cstar.testcase.TeamCalendar"/>
  <testcase name="TC07" time="1.801" classname="com.vsii.cstar.testcase.TeamCalendar"/>
  <testcase name="TC08" time="0.001" classname="com.vsii.cstar.testcase.TeamCalendar">
    <skipped/>
  </testcase> <!-- TC08 -->
  <testcase name="TC08" time="0.001" classname="com.vsii.cstar.testcase.TeamCalendar">
    <skipped/>
  </testcase> <!-- TC08 -->
  <testcase name="TC08" time="0.001" classname="com.vsii.cstar.testcase.TeamCalendar">
    <skipped/>
  </testcase> <!-- TC08 -->
  <testcase name="TC09" time="0.001" classname="com.vsii.cstar.testcase.TeamCalendar">
    <skipped/>
  </testcase> <!-- TC09 -->
</testsuite> <!-- TestTeamCalendar -->
